src/pybind/mgr/dashboard/frontend/src/app/shared/components/language-selector/supported-languages.enum.ts

```diff
@@ -6,5 +6,6 @@ export enum SupportedLanguages {
   'id-ID' = 'Bahasa Indonesia',
   'pt-PT' = 'Português',
   'es-ES' = 'Español',
-  'zh-CN' = '中文'
+  'zh-CN' = '中文', 
+  'pl-PL' = 'Polish'
```

Comment:
```suggestion
  'pl-PL ' = 'Polski'
```

i think the comment from @tspmelo https://github.com/ceph/ceph/pull/26907#discussion_r264641671 still applies